### PR TITLE
﻿Update docker files

### PR DESCRIPTION
**Pull request by sgessa**
# Setup

## Configure ENV

- Copy `.env.example` to `.env`
- Set `MS2_DATA_FOLDER` and rename it to `MS2_DOCKER_DATA_FOLDER`

## Setup Database and Seed Data

```sh
docker compose run --rm -it db-seed
```

## Ingest Metadata

```sh
docker compose run --rm file-ingest
```

## Run servers

```sh
docker compose up login world game-main
```

### CODE REVIEW

File: compose.yml [hunk 1]
```diff
@@ -1,81 +1,155 @@
...
+      - 20001:20001
+    links:
+      - world
+    env_file:
+      - .env
     environment:
+      DB_IP: mysql
+      WORLD_SERVICE: world
       GAME_DB_CONNECTION: "Server=host.docker.internal;Database=game-server;User=root;Password=maplestory;oldguids=true"
       DATA_DB_CONNECTION: "Server=host.docker.internal;Database=maple-data;User=root;Password=maplestory;oldguids=true"
```
Comment (AngeloTadeucci) [inline compose.yml:58]: i dont think this url works anymore, you need each separate variable. change login, world, game and web containers

DB_IP=localhost
DB_PORT=3306
DB_USER=root
DB_PASSWORD=maplestory

DATA_DB_NAME=maple-data
GAME_DB_NAME=game-server

File: compose.yml [hunk 2]
```diff
@@ -1,81 +1,155 @@
...
+      - 21002:21002
+    env_file:
+      - .env
     environment:
+      DB_IP: "mysql"
       GAME_DB_CONNECTION: "Server=host.docker.internal;Database=game-server;User=root;Password=maplestory;oldguids=true"
       DATA_DB_CONNECTION: "Server=host.docker.internal;Database=maple-data;User=root;Password=maplestory;oldguids=true"
       GAME_CHANNEL: 1
       GAME_PORT: 20002
       GRPC_CHANNEL_PORT: 21002
```
Comment (AngeloTadeucci) [inline compose.yml:98]: these can be removed on both game containers

**Review by AngeloTadeucci — CHANGES_REQUESTED** [review]

File: compose.yml (hunk 2 above)
Comment (sgessa) [reply compose.yml:98]: @AngeloTadeucci are you sure?

They are used here:
https://github.com/AngeloTadeucci/Maple2/blob/14703254b9a4a41e73d21156ec32214c1f164983/Maple2.Server.Core/Constants/Target.cs#L30-L45

**Comment by sgessa** [comment]
@AngeloTadeucci please see PR description

File: compose.yml (hunk 2 above)
Comment (AngeloTadeucci) [reply compose.yml]: it was removed in this commit: https://github.com/AngeloTadeucci/Maple2/commit/41f7a7f78fcbf5c6846fd1bd5e637a23af6cc16d

i updated the base branch

**Review by AngeloTadeucci — APPROVED** [review]

**Pull request merged**